### PR TITLE
fix: collateral should be flagged false when got liquidated totally

### PR DESCRIPTION
**Pull request by GopherJ**
vars.liquidationProtocolFee has been [removed](https://github.com/aave/aave-v3-core/blob/f3e037b3638e3b7c98f0c09c56c5efde54f7c5d2/contracts/protocol/libraries/logic/LiquidationLogic.sol#L530) from vars.actualCollateralToLiquidate, but the whole `vars.liquidationProtocolFee + vars.actualCollateralToLiquidate` is going to be liquidated. So we are missing adding it back [here](https://github.com/aave/aave-v3-core/blob/f3e037b3638e3b7c98f0c09c56c5efde54f7c5d2/contracts/protocol/libraries/logic/LiquidationLogic.sol#L203)

I cannot access `@aave/deploy-v3` so I made some assumptions:

WETH:
 - liquidation threshold = 85%
 
USDC:
 - liquidation threshold = 90%
 - price = 0.0005 WETH

DAI:
 - liquidation threshold = 90%
 - price = 0.0005 WETH

I'll update the test later according to real data

DAI

### CODE REVIEW

File: test-suites/liquidation-edge.spec.ts [hunk 1]
```diff
@@ -214,4 +215,126 @@ makeSuite('Pool Liquidation: Edge cases', (testEnv: TestEnv) => {
     expect(isBorrowing(userConfigBefore, daiData.id)).to.be.true;
     expect(isBorrowing(userConfigAfter, daiData.id)).to.be.false;
   });
+
+  it('Liquidate the whole WETH collateral with 10% liquidation fee, asset should not be set as collateralized anymore', async () => {
+    const { pool, users, dai, usdc, weth, aWETH, oracle, configurator } = testEnv;
+
+    configurator.setLiquidationProtocolFee(weth.address, "1000"); // 10%
```
Comment (miguelmtzinf) [inline test-suites/liquidation-edge.spec.ts:222]:
```suggestion
    await configurator.setLiquidationProtocolFee(weth.address, "1000"); // 10%
```

**Comment by miguelmtzinf** [comment]
Merging to my own PR so I can fix the test case provided. Thanks a lot @GopherJ
Highly appreciate your contribution!
